sources/packages/backend/apps/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -94,6 +95,21 @@ export class InstitutionLocationInstitutionsController extends BaseController {
       );
     }
 
+    const isInstitutionCodeDuplicate =
```

Comment:
I would recommend moving this method to the service layer within the method `saveLocation` since we must ensure this condition all the time. From the `saveLocation` a CustomNamed error can be thrown and captured here also.

Reply:
ok. Done. Thanks!